### PR TITLE
[CCXDEV-8782] Add the new types

### PR DESCRIPTION
**Pull request by juandspy**
# Description

I'm splitting https://github.com/RedHatInsights/ccx-notification-service/pull/243 into smaller pieces. This one adds the `event_target_id` value in the database as a new type, instead of using an int.

Part of #CCXDEV-8782

## Type of change

- Refactor (refactoring code, removing useless files)

## Testing steps

BDD testing + unit testing.

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -58,13 +59,20 @@ type NotificationTypeID int
 // StateID represents ID value in `states` table.
 type StateID int
 
+// EventTarget matches the backend int code in the database where the notifications are sent (CCXDEV-8767)
+type EventTarget int8
```
Comment (tisnik) [inline types/types.go:63]: uint8 probably?

**Review by tisnik — APPROVED** [review]
Nice example of minimalist PR!

File: types/types.go (hunk 1 above)
Comment (juandspy) [reply types/types.go:63]: I thought about using that type, but I'm not sure. Maybe we use `-1` for something in the future. I can put `uint8` there, not a big deal. What should I do?

Comment (tisnik) [reply types/types.go:63]: really does not matter :-)

**Review by epapbak — APPROVED** [review]
+1

**Pull request merged**